canopy/canopyjs/src/transactions/batch/errors/index.ts

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
MissingFieldError can probably be defined here to reduce the import nesting.

Reply:
This will cause lint errors later on if any other error types get added